stdlib/2and3/binascii.pyi

```diff
@@ -15,31 +15,31 @@ else:
     _Bytes = bytes
     _Ascii = Union[bytes, str]
 
-def a2b_uu(string: _Ascii) -> bytes: ...
+def a2b_uu(__data: _Ascii) -> bytes: ...
```

Comment:
This module wins at having the most cryptic function names

Reply:
Yeah, I leaned heavily on line numbers to make these changes